.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install:
+  - "pip install --upgrade setuptools"
+  - "pip install --upgrade pip"
+  - "pip install -r requirements.txt"
+# command to run tests
+script:
+  - cd ..
+  - wget https://github.com/DIRACGrid/DIRAC/archive/rel-v6r15.zip
+  - unzip -q rel-v6r15.zip
+  - mv DIRAC-rel-v6r15 DIRAC
```

Comment:
I don't like hard-coding a version of DIRAC. If really it's need to do something like that, than you should use the content of "master" which corresponds to the "last production version". So I would probably use https://github.com/DIRACGrid/DIRAC/archive/master.zip